### ISSUE TITLE
MinifyHtml Transformer breaks line wraps

### ISSUE THREAD

**Issue by swissspidy** [ampproject/amp-toolbox-php#427]
In the Web Stories WordPress plugin we use `white-space: pre-wrap;` for treating newlines in paragraphs.

So HTML markup like this:

```html
<span>Foo
Bar
Baz

Foobar

Barbaz</span>
```

Will be displayed like this:
![Screenshot 2021-12-01 at 21 02 08](https://user-images.githubusercontent.com/841956/144306945-53dd7660-f381-4221-9ada-999cc9a98614.png)

Unfortunately, the `MinifyHtml` transformer turns this HTML into 

```html
<span>Foo
Bar
Baz Foobar Barbaz</span>
```

and thus breaks the newlines like this:

![Screenshot 2021-12-01 at 21 02 15](https://user-images.githubusercontent.com/841956/144307076-b959a547-2719-4e41-bf5d-7f7b08425ff8.png)

**Comment by westonruter** [comment]
Good catch. I don't see a way around this other than to turn off the transformer by default.
